napari/utils/settings/_defaults.py

```diff
@@ -394,4 +394,39 @@ class NapariConfig:
         preferences_exclude = ['schema_version', 'disabled_plugins']
 
 
-CORE_SETTINGS = [AppearanceSettings, ApplicationSettings, PluginsSettings]
+class ExperimentalSettings(BaseNapariSettings):
+    """Developer/Experimental Settings."""
+
+    schema_version: SchemaVersion = (0, 1, 1)
+
+    NAPARI_OCTREE: bool = Field(
+        True,
+        title=trans._("NAPARI_OCTREE"),
+        description=trans._("Use OctreeImage instead of Image."),
+    )
+
+    NAPARI_ASYNC: bool = Field(
```

Comment:
The name should be `async_`  and we need an alias as `async` if possible

Reply:
Custom environment variable names can be set in two ways:
Config.fields['field_name']['env'] (see auth_key and redis_dsn above)
Field(..., env=...) (see api_key above)

https://pydantic-docs.helpmanual.io/usage/settings/

We need to use `env="napari_async"` or `env="async"` we need to test which one works

Reply:
Yeah, I see what you mean.  I guess I was thinking that for anyone who was setting this via environment variable would more likely recognize it if it looked the same as setting the environment variable.  But, if they know to change it, they will get it.

Reply:
>"NAPARI_OCTREE" or "OCTREE" or "octree"?

I do not think we need to show the exact env names. @sofroniewn ?

`title ="Use octree"` `description="??"`
`title ="Use async mode"` `description="??"`

Reply:
Yeah agreed - I might actually prefer not to even use the word octree here, it's not actually accurate to what is implemented and most users wont know what it is. I can add some suggestions here in the morning. Maybe @jni has ideas too

Reply:
@sofroniewn @jni 

I made an attempt at descriptive names.  What do you think?

<img width="949" alt="Screen Shot 2021-05-27 at 1 40 36 PM" src="https://user-images.githubusercontent.com/54282105/119879837-43dec200-bef1-11eb-9b4b-0da4cbecd627.png">

Reply:
Loving it! I suggest that we add `asynchronous` in front of `tiling` as well, to make it clear that it is asynchronous tiles, we can refine further later - but this looks like a great start!

Reply:
all these should be false by default